R/guides-none.r

```diff
@@ -0,0 +1,44 @@
+
+#' Empty guide
+#'
+#' This guide draws nothing.
+#'
+#' @inheritParams guide_axis
+#'
+#' @export
+#'
+guide_none <- function(title = waiver(), position = waiver()) {
+  structure(
+    list(
+      title = title,
+      position = position,
+      available_aes = "any"
+    ),
+    class = c("guide", "guide_none")
```

Comment:
Is there a reason why this guide doesn't follow the naming of the other guides? shouldn't the class be. `c('guide', 'none')`

Reply:
It's not clear to me what we should do in this situation where the previous code is "wrong" in some sense. (Although here it should be `c("guide_none", "guide")`).

Reply:
We could elect to fix it once and for all when we finally move guides from S3 to ggproto

Reply:
In other words, should we stay consistent with the past (i.e. the previous S3 class naming scheme which we know now is suboptimal) and hence be inconsistent with current naming schemes, or should be consistent with current naming schemes introducing internal inconsistencies? (I think the answer is different to (e.g.) the argument naming scheme because this is internal, not user facing)

Reply:
I think we should stay consistent for the time being and update them all in one fell swoop during a larger rewrite

Reply:
let's just keep it for now... your reasons seem sensible